c2corg_api/models/image.py

```diff
@@ -1,32 +1,63 @@
 from sqlalchemy import (
+    Boolean,
     Column,
+    Float,
+    ForeignKey,
     Integer,
     SmallInteger,
-    ForeignKey,
-    Enum
+    String
     )
 
 from colanderalchemy import SQLAlchemySchemaNode
 
-from c2corg_api.models import schema
-from c2corg_api.models.utils import copy_attributes
+from c2corg_api.models import schema, enums
+from c2corg_api.models.utils import copy_attributes, ArrayOfEnum
 from c2corg_api.models.document import (
     ArchiveDocument, Document, get_update_schema, geometry_schema_overrides,
     schema_document_locale, schema_attributes)
-from c2corg_common.attributes import activities
 
 IMAGE_TYPE = 'i'
 
 
 class _ImageMixin(object):
-    activities = Column(
-        Enum(name='activities', inherit_schema=True, *activities),
-        nullable=False)
+
+    activities = Column(ArrayOfEnum(enums.activity_type))
+
+    categories = Column(ArrayOfEnum(enums.image_category))
+
+    image_type = Column(enums.image_type)
+
+    author = Column(String(100))
+
+    # FIXME not sure still needed:
```

Comment:
> In v6, since we no longer support old browsers, I think we will be able to display the SVG file directly? 

Yes, we should be able to directly use the SVG files. 

> What about the resized/cropped versions?

You mean if we also should create thumbnail versions for SVG files? If I look at the example file that you have linked to, it would make sense.